video_xblock/static/js/studio-edit.js

```diff
@@ -626,29 +615,6 @@ function StudioEditableXBlock(runtime, element) {
         return url;
```

Comment:
We still need it [here](https://github.com/raccoongang/xblock-video/blob/refactor/default_transcripts/video_xblock/static/js/studio-edit.js#L633) and [here](https://github.com/raccoongang/xblock-video/blob/refactor/default_transcripts/video_xblock/static/js/studio-edit.js#L685)